packages/gestalt/src/TableOfContents/TableOfContentsItemList.js

```diff
@@ -0,0 +1,22 @@
+// @flow strict
+import { type Node } from 'react';
+import classNames from 'classnames';
+import getChildrenToArray from './getChildrenToArray.js';
+import { NestingProvider } from '../contexts/NestingProvider.js';
+import Layout from '../Layout.css';
+import Whitespace from '../Whitespace.css';
+
+type Props = {|
+  children: Node,
+|};
+
+export default function TableOfContentsItemList({ children }: Props): Node {
+  const ulClassNames = classNames(Layout.flex, Layout.flexColumn, Whitespace.m0, Whitespace.p0);
+  const childrenArray = getChildrenToArray({ children });
```

Comment:
could we move this logic to TableOfContents.js?

any reason to create separate file TableOfContentsItemList.js?

just for readability

Reply:
OH I see we are reusing it... disregard!